Source/JavaScriptCore/API/JSContextRef.cpp

```diff
@@ -249,6 +249,34 @@ void JSGlobalContextSetName(JSGlobalContextRef ctx, JSStringRef name)
     globalObject->setName(name ? name->string() : String());
 }
 
+bool JSGlobalContextIsInspectable(JSGlobalContextRef ctx)
+{
+    if (!ctx) {
+        ASSERT_NOT_REACHED();
+        return false;
+    }
+
+    JSGlobalObject* globalObject = toJS(ctx);
+    VM& vm = globalObject->vm();
+    JSLockHolder lock(vm);
+
+    return globalObject->remoteDebuggingEnabled();
```

Comment:
hmm why not also rename this to `inspectable`?  I thought the plan was to have `inspectable` control _all_ inspection, not just remote inspection?